modules/ROOT/partials/index-pages/premium-plugins.adoc

```diff
@@ -33,6 +33,12 @@ xref:advanced-typography.adoc[Advanced Typography]
 
 Replace common typewriter-style characters with their professional typesetter equivalents.
 
+a|
+[.lead]
+xref:ai.adoc[AI Assistant]
+
+Send queries to registered AI APIs and set the results in the document or temporarily stores the results for later use.
```

Comment:
@MitchCTiny placeholder, if we decide to update the previous comment regarding this statement.

Reply:
See [this commit](https://github.com/tinymce/tinymce-docs/commit/de1c7127e31ef6158e2d9fdbb82b7468321db76b) for the re-write that covers this.